CYCLONEDDS_QUALITY_DECLARATION.md

```diff
@@ -1,150 +1,210 @@
 
 ### Linters and Static Analysis [4.v]
 
-There is no explicit documentation about the use of sanitizers in `CycloneDDS`, but it appears that the `USE_SANITIZER` build option can be used to enable address and thread sanitizers.
-In continuous integration, ASAN is enabled for some of the test matrix.
+`rmw_cyclonedds` uses and passes all the ROS2 standard linters and static analysis tools for a C++ package as described in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#linters-and-static-analysis). Passing implies there are no linter/static errors when testing against CI of supported platforms.
+
+Cyclone DDS has automated daily [Synopsys Coverity static code analysis](https://www.synopsys.com/software-integrity/security-testing/static-analysis-sast.html) with public results that can be seen [here](https://scan.coverity.com/projects/eclipse-cyclonedds-cyclonedds). Cyclone DDS defect density is 0.05 per 1,000 lines of code as of Aug 11th 2020. Omitting the Java idlc which is **not** used by ROS 2 gives 0.03 per 1,000 lines of code. For comparison the average defect density of open source software projects of similar size is 0.5 per 1,000 lines of code.
 
-There do not appear to be any linters enabled for the `CycloneDDS` repository.
+There are no linters enabled for the Cyclone DDS repository.
+
+Coding style is considered and enforced in the review of all CyconeDDS pull requests.
+Contributors are regularly asked to rewrite or reformat their code contributions before pull requests are accepted. 
+This is a matter of public record in the CycloneDDS pull request history.  
 
 ## Dependencies [5]
 
 ### Direct Runtime ROS Dependencies [5.i]
 
-As an external dependency, there are no ROS dependencies in `CycloneDDS`.
+As an external dependency, there are no ROS dependencies in Cyclone DDS.
 
 ### Optional Direct Runtime ROS Dependencies [5.ii]
 
-As an external dependency, there are no ROS dependencies in `CycloneDDS`.
+As an external dependency, there are no ROS dependencies in Cyclone DDS.
 
 ### Direct Runtime non-ROS Dependency [5.iii]
 
-The only runtime dependency of `CycloneDDS` is `OpenSSL`, a widely-used secure communications suite.
-If `CycloneDDS` is built without security enabled, the product has no apparent runtime dependencies.
+The only runtime dependency of Cyclone DDS is [OpenSSL](https://openssl.org), a widely-used secure communications suite.
+If Cyclone DDS is built without security enabled, the product has no runtime dependencies.
+
+[Eclipse Zenoh](http://zenoh.io) is optional non-ROS dependency of Cyclone DDS providing: scalable, fault tolerant DDS discovery with "need to know" discovery matching of endpoints; ROS2 DDS to ROS2 DDS routing across a mesh of Zenoh nodes; ROS 2 DDS to ROS 2 DDS bridging across wireless WiFi, LoRa, LTE, 5G, satellite; ROS 2 DDS routing to/from cloud; microcontroller support; `rmw_zenoh` for microROS drop-in replacment for DDS-XRCE.
```

Comment:
No: Cyclone doesn't in any require the presence of Zenoh. That it can cooperate with it is very nice (well, more than just "nice", really) but that doesn't make it a dependency.